src/common/transformations/src/transformations/op_conversions/detection_output_upgrade.cpp

```diff
@@ -48,13 +48,13 @@ pass::ConvertDetectionOutput1ToDetectionOutput8::ConvertDetectionOutput1ToDetect
 
         std::shared_ptr<opset8::DetectionOutput> detection_output_v8_node = nullptr;
         if (detection_output_v1_node->get_input_size() == 3) {
-            auto detection_output_v8_node =
+            detection_output_v8_node =
```

Comment:
It should be covered by transformation tests. Actually transformation was not working and issue was detected during enabling of transformation in the plug-in.

Reply:
I will investigate it

Reply:
@nshchego, investigated. There is a gap in type_info comparator used in transformations tests. I emailed about this bug in detail. For now, I use more advanced compator to check attributes as well which number is changed from v1 to v8 opset. So now the transformation tests are more reliable.